specifyweb/context/app_resource.py

```diff
@@ -22,13 +22,31 @@
 DIR_LEVELS = ['Personal', 'UserType', 'Collection', 'Discipline', 'Common', 'Backstop']
 
 # At the discipline level of the hierarchy, filesystem resources are found in
-# the directories defined in "disciplines.xml".
-disc_file = os.path.join(settings.SPECIFY_CONFIG_DIR, "disciplines.xml")
-
-disciplines = ElementTree.parse(disc_file)
-
-discipline_dirs = dict( (disc.attrib['name'], disc.attrib.get('folder', disc.attrib['name']))
-    for disc in disciplines.findall('discipline') )
+# the directories were defined in "disciplines.xml" but are now defined here.
+DISCIPLINE_DIRS = {
+    "fish": "fish",
+    "herpetology": "herpetology",
+    "paleobotany": "invertpaleo",
+    "invertpaleo": "invertpaleo",
+    "vertpaleo": "vertpaleo",
+    "bird": "bird",
+    "mammal": "mammal",
+    "insect": "insect",
+    "botany": "botany",
+    "invertebrate": "invertebrate",
+    "minerals": "minerals",
+    "anthropology": "anthropology",
+    # "vascplant": "vascplant",
```

Comment:
was this commented out intentionally?

Reply:
Since vascular plants form definitions are still being retrieved, would that point to this code not functioning as expected? Why are minerals included at this time when no form definition exists?

Reply:
Here are lines 11-24 in the `disciplines.xml` file in Specify 6 ([link](https://github.com/specify/specify6/blob/945d28f00288d2d64979848fb2fb63fda34bdd1b/config/disciplines.xml#L11-L25)):

```xml
    <discipline name="fish"         title="Ichthyology"  abbrev="FISH" isembedce="false" type="0"/>
    <discipline name="herpetology"  title="Herpetology"  abbrev="HRP"  isembedce="true" type="0"/>
    <discipline name="paleobotany"  title="Paleobotany"  abbrev="PLB"  folder="invertpaleo" isembedce="true" type="0"/>
    <discipline name="invertpaleo"  title="Invertebrate Paleontology"  abbrev="IVTP" isembedce="true" type="0"/>
    <discipline name="vertpaleo"    title="Vertebrate Paleontology"    abbrev="VRTP" isembedce="true" type="0"/>
    <discipline name="bird"         title="Ornithology"  abbrev="ORN"  isembedce="true" type="0"/>
    <discipline name="mammal"       title="Mammalogy"    abbrev="MAM"  isembedce="true" type="0"/>
    <discipline name="insect"       title="Entomology"   abbrev="ENTO" isembedce="true" type="0"/>
    <discipline name="botany"       title="Botany"       abbrev="BNTY" isembedce="true" type="0"/>
    <discipline name="invertebrate" title="Invertebrate Zoology" abbrev="IVTZ" isembedce="true" type="0"/>
    <!--  <discipline name="vascplant"    title="Vascular Plants"     abbrev="VP" folder="botany" isembedce="true"    type="0"/>  -->
    <discipline name="minerals"     title="Mineralogy"   abbrev="MIN"  isembedce="true" type="4"/>
    <!--  <discipline name="fungi"        title="Mycology"     abbrev="MYCO"  folder="botany" isembedce="true" type="0"/> -->
    <discipline name="anthropology" title="Anthropology" abbrev="ANTH" isembedce="true" type="3"/>
```

It appears that this is a direct translation, so I guess we're OK. Would be worth revisiting the significance of this since `vascplant` _is_ a valid discipline and `anthropology` is not.

Reply:
Ya, it was a direct translation of the file.  I'm going to leave it as is for now.  I could comment out more of the disciplines as long as we are sure it will not affect other code.